README.md

```diff
@@ -54,8 +59,8 @@ The following command imports a collection from the `data/import/my_exhibition`
 import-folder --config=config.json --path=data/import/my_exhibition --name=demo
 ```
 
-Since the images themselves are not stored in the database, VREP will maintain a folder containing the exhibition images
-in `data/` (relative to VREM).
+Since the image and ambient audio files are not stored in the database, VREP will maintain a folder containing the
```

Comment:
Ambient audio is not supported anymore in the data model?

Reply:
Of course, but the files themselves are not stored in MongoDB (only their path, as before), so they're copied into the data folder specified in the configuration upon exhibition import. With this, VREM can serve all previously imported exhibitions instead of a single exhibition.